### ISSUE TITLE
celery.signature should accept str

### ISSUE THREAD

**Issue by nicolaerosia** [sbdchd/celery-types#8]
first of all, thank you for creating this package!

I have an issue with `celery.signature` not accepting `str`:
```
error: Argument 1 to "signature" has incompatible type "str"; expected "Union[Signature, Dict[str, Any]]"  [arg-type]
```

The celery documentation shows an example with `str`, https://docs.celeryproject.org/en/stable/userguide/canvas.html
Which contradicts the source code comments here: https://github.com/celery/celery/blob/5d72aeedb6329b609469c63998e9335e017bd204/celery/canvas.py#L1568

You can see that `Signature` which is called by `signature` supports `str` here: https://github.com/celery/celery/blob/5d72aeedb6329b609469c63998e9335e017bd204/celery/canvas.py#L149

**Comment by sbdchd** [comment]
Thanks for the report, looks like we needs to adjust: https://github.com/sbdchd/celery-types/blob/5e8ec301eaaefbaecc05710eaeff5eabd126ec9b/typings/celery/canvas.pyi#L364

**Comment by nicolaerosia** [comment]
yep, probably it's worth making a PR against celery as well to update the comment there

**Comment by sbdchd** [comment]
Should be fixed in 0.3.1, thanks again for the bug report!
